documentation/get-started/certification.en.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Certification
+tx_slug: documentation_get-started_certifiation
+---
+
+# Certification
+
+The QField Certificate Program is designed to foster community involvement in the QField project and provide high-quality education for the QField application. Therefore, the application process requires applicants to both detail their contributions to the QField project and make their training materials available for review. Activities that contribute to the QField project include:
+
+- Contributions to the QField documentation and training materials
+
+- Translation of QField materials
+
+- Filing bug reports
+
+- Financially supporting new feature development
+
+- Financial contributions to and sponsorship of QField
+
+- Involvement in your local QField User Group
+
+- Publication of open QField course ware
+
+To contact the certification team, please write to info@opengis.ch.
+
+## Certification Process
+
+Each of your courses requires you to provide training material. This includes exercises, lectures, and data. This material will be reviewed to ensure that it is current, accurate, and of high quality. If the material is not of adequate quality, it may be refused.
```

Comment:
```suggestion
Each of your courses requires a check of the training material. This includes exercises, lectures, and data. This material will be reviewed to ensure that it is current, accurate, and of high quality. If the material is not of adequate quality, it may be refused.
```